driver/dotnet/stress.py

```diff
@@ -26,6 +28,7 @@
         cmd.append("--filter")
         cmd.append("DisplayName~SingleInstanceStressTests")
 
+    # Temp commented out to test time to run on TeamCity
     subprocess.run(cmd, universal_newlines=True,
-                   stderr=subprocess.STDOUT, check=True,
-                   cwd="Neo4j.Driver/Neo4j.Driver.Tests.Integration")
+                  stderr=subprocess.STDOUT, check=True,
```

Comment:
Nitpick... The removed space will at least cause my pep8 checker to warn on those lines